app/api/groups/update_permissions.go

```diff
@@ -73,7 +73,7 @@ type managerGeneratedPermissions struct {
 //     of `{source_group_id}` which should be an ancestor of the `{group_id}`.
 //
 //   * The user giving the access must have `permissions_generated.can_grant_view` >= given `can_view`
-//     for the item.
+//     for the item. If he does not, the item must be a root activity/skill for an ancestor group of the user.
```

Comment:
🤨 Isn't the remark about the root related with the next line instead?

Reply:
Right. Fixed :(